custom_components/blnet/switch.py

```diff
@@ -114,15 +126,17 @@ def is_on(self):
 
     def turn_on(self, **kwargs):
         """Turn the device on."""
-        self.communication.turn_on(self._id)
-        self._state = STATE_ON
-        self._assumed_state = True
+        if self._mode != 'AUTO':
```

Comment:
I see your intention. However I decided to make it possible to manually override automatically driven switches. The reason is that waiting for the status update after first turning the switch from "auto" to "hand" to then change the current state too cumbersome.

Reply:
Of cause, that's right. Best would be to have a control with 3 states in HA. I had a look at entity-type input_select, but that's not very comfortable. Maybe I'll try this extention one day:
https://github.com/thomasloven/lovelace-slider-entity-row

Reply:
Yeah that would be my preferred option too... If you get any usefuel results let me know :)